Cargo.toml

```diff
@@ -108,13 +109,8 @@ cairo-vm = { git = "https://github.com/Moonsong-Labs/cairo-vm", branch = "notles
 ] }
 
 # Snos & Sharp (Starkware)
-# TODO: Update branch to main once the PR is merged (PR #368 in the snos repository)
```

Comment:
I think it's high time that we include a TODO-to-issue workflow, because such changes will not be remembered for long.

Reply:
is this a common thing? haven't seen it before

Reply:
also, in this specific case, we're deleting a TODO btw, not adding one but your point probably stays. although, i do know there are extensions that people normally use to see the complete list of TODOs in the code

Reply:
Yea there are extensions and I personally use them, but I have seen maintainers become more active about a TODO when it's converted to an issue, maybe because it get's shown up in the issues tab which is usually tracked

here's one to try : https://github.com/marketplace/actions/todo-to-issue

Reply:
created issue